src/main/java/com/crowdin/client/tasks/model/CreateLanguageServiceTaskRequest.java

```diff
@@ -5,11 +5,9 @@
 
 @Data
 @EqualsAndHashCode(callSuper = true)
-public class CreateLanguageServiceTaskRequest extends AddTaskRequest {
+public class CreateLanguageServiceTaskRequest extends AddTaskFilesBasedRequest {
 
-    private Type type;
+    private TypeVendor type;
     private String vendor;
-    private Boolean skipUntranslatedStrings;
```

Comment:
We can't just remove the deprecated API properties because they might be used somewhere. In particular, in this case, it is being used in the Crowdin CLI - https://crowdin.github.io/crowdin-cli/commands/crowdin-task-add

I would suggest marking it as deprecated but keeping it in the API client until the next major release. Please apply this to all deprecated task properties in the API docs.

Reply:
Updated the solution.